### PR TITLE
[BEAM-7776] Remove Pkb from file-based IOIT jobs

### PR DESCRIPTION
**Pull request by lgajowy**
See https://issues.apache.org/jira/browse/BEAM-7772 for more context

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

Post-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

Lang | SDK | Apex | Dataflow | Flink | Gearpump | Samza | Spark
--- | --- | --- | --- | --- | --- | --- | ---
Go | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go/lastCompletedBuild/) | --- | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go_VR_Flink/lastCompletedBuild/) | --- | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go_VR_Spark/lastCompletedBuild/)
Java | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Apex/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Flink_Batch/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Flink_Streaming/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Gearpump/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Samza/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Spark/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Spark_Batch/lastCompletedBuild/)
Python | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python2/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Python35/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Python36/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Python37/lastCompletedBuild/) | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_VR_Dataflow/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Py_ValCont/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Python_PVR_Flink_Cron/lastCompletedBuild/) | --- | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_VR_Spark/lastCompletedBuild/)

Pre-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

--- |Java | Python | Go | Website
--- | --- | --- | --- | ---
Non-portable | [Build Status](https://builds.apache.org/job/beam_PreCommit_Java_Cron/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Python_Cron/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Go_Cron/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Website_Cron/lastCompletedBuild/) 
Portable | --- | [Build Status](https://builds.apache.org/job/beam_PreCommit_Portable_Python_Cron/lastCompletedBuild/) | --- | ---

See [.test-infra/jenkins/README](https://github.com/apache/beam/blob/master/.test-infra/jenkins/README.md) for trigger phrase, status and link of all Jenkins jobs.

### CODE REVIEW

**Comment by lgajowy** [comment]
Run seed job

**Comment by lgajowy** [comment]
Run seed job

**Comment by lgajowy** [comment]
Run Java TextIO Performance Test

**Comment by lgajowy** [comment]
Run seed job

**Comment by lgajowy** [comment]
Run Java TextIO Performance Test

**Comment by lgajowy** [comment]
Run seed job

**Comment by lgajowy** [comment]
Run Java TextIO Performance Test

**Comment by lgajowy** [comment]
Run seed job

**Comment by lgajowy** [comment]
Run Java ParquetIO Performance Test

**Comment by lgajowy** [comment]
@aromanenko-dev could you take a look?

**Comment by aromanenko-dev** [comment]
@lgajowy Yes, I'll do

File: .test-infra/jenkins/job_PerformanceTests_FileBasedIO_IT.groovy [hunk 1]
```diff
@@ -16,155 +16,136 @@
...
         [
-                jobName           : 'beam_PerformanceTests_TextIOIT',
-                jobDescription    : 'Runs PerfKit tests for TextIOIT',
-                itClass           : 'org.apache.beam.sdk.io.text.TextIOIT',
-                bqTable           : 'beam_performance.textioit_pkb_results',
-                prCommitStatusName: 'Java TextIO Performance Test',
-                prTriggerPhase    : 'Run Java TextIO Performance Test',
-                extraPipelineArgs: [
+                name               : 'beam_PerformanceTests_TextIOIT',
+                description        : 'Runs PerfKit tests for TextIOIT',
```
Comment (aromanenko-dev) [inline .test-infra/jenkins/job_PerformanceTests_FileBasedIO_IT.groovy]: Here and all below - don't we need to change `PerfKit` name to something else? Maybe just `performance`?

Comment (lgajowy) [reply .test-infra/jenkins/job_PerformanceTests_FileBasedIO_IT.groovy]: I agree - I missed this. Thanks!

**Comment by lgajowy** [comment]
> Also, would you mind to squash all commits into one?

The reason I keep 3 commits here is because they do 3 different things here. One does the actual change. The second cleans up naming and does only that. The third applies formatting rules. 

If I squash them this will be a blob of changes very hard to read (practically every line get's touched). I prefered not to do this for the sake of having clean git history here.

**Comment by lgajowy** [comment]
Run seed job

**Comment by lgajowy** [comment]
Run Java TextIO Performance Test

**Comment by lgajowy** [comment]
Run Java CompressedTextIO Performance Test

**Comment by lgajowy** [comment]
Run Java ManyFilesTextIO Performance Test

**Comment by lgajowy** [comment]
Run Java AvroIO Performance Test

**Comment by lgajowy** [comment]
Run Java TFRecordIO Performance Test

**Comment by lgajowy** [comment]
Run Java XmlIO Performance Test

**Comment by lgajowy** [comment]
Run Java ParquetIO Performance Test

**Comment by lgajowy** [comment]
@aromanenko-dev all potentially affected tests have passed. Could you take another look?

**Comment by lgajowy** [comment]
I posted the reason here (in case you missed this, lot's of comments): https://github.com/apache/beam/pull/9287#issuecomment-520377808

**Comment by aromanenko-dev** [comment]
Yes, my bad, I missed that with other `Run smth` commits =) 
So, it's fine

**Pull request merged**